docs/concepts/hashing.md

```diff
@@ -49,15 +48,17 @@ Cryptographic hashes come with a couple of very important characteristics:
 - **unique** - it's infeasible to generate the same hash from two different messages
 - **one-way** - it's infeasible to guess or calculate the input message from its hash
 
-These features also mean we can use a cryptographic hash to identify any piece of data: the hash is unique to the data we calculated it from and it's not too long so sending it around the network doesn't take up a lot of resource. A hash is a fixed length, so the SHA-256 hash of a one-gigabyte video file is still only 32 bytes. 
+These features also mean we can use a cryptographic hash to identify any piece of data: the hash is unique to the data we calculated it from and it's not too long so sending it around the network doesn't take up a lot of resource. A hash is a fixed length, so the SHA-256 hash of a one-gigabyte video file is still only 32 bytes.
 
-That's critical for a distributed system like IPFS, where we want to be able to store and retrieve data from many places. A computer running IPFS can ask all the peers it's connected to whether they have a file with a particular hash and, if one of them does, they send back the whole file. Without a short, unique identifier like a cryptographic hash, that wouldn't be possible. This technique is called [content addressing](content-addressing.md) — because the content itself is used to form an address, rather than information about the computer and disk location it's stored at.
+That's critical for a distributed system like IPFS, where we want to be able to store and retrieve data from many places. A computer running IPFS can ask all the peers it's connected to whether they have a file with a particular hash and, if one of them does, they send back the whole file. Without a short, unique identifier like a cryptographic hash, this kind of [content addressing](content-addressing.md) wouldn't be possible.
```

Comment:
I'm not a native speaker, so it might just be me. I would say there are different kinds of "addressing" (content vs. location based), but there is only one kind of "content addressing". To me it reads like there would be different kinds of content addressing.

Reply:
@vmx It's kind of slangy, but you have a point, we should make it work for an international audience.